proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -88,26 +75,12 @@ message CuratedPluginSearchResult {
   bool deprecated = 4;
 }
 
-message TemplateSearchResult {
-  option deprecated = true;
-  string id = 1;
-  string name = 2;
-  // The name of the user or organization
-  // who is the owner of this template.
-  string owner = 3;
-  // The visibility of the template.
-  PluginVisibility visibility = 4;
-  bool deprecated = 5;
-}
-
 message SearchResult {
   oneof item {
     RepositorySearchResult repository = 1;
     OrganizationSearchResult organization = 2;
     UserSearchResult user = 3;
     TeamSearchResult team = 4;
-    PluginSearchResult plugin = 5 [deprecated = true];
```

Comment:
I reserved the event fields since they are stored. But didn't add reserved for any of the others since the BSR should be the sole consumer of these. But opinions may vary though on how much of the legacy stuff we want to break.